libraries/AP_Scripting/docs/docs.lua

```diff
@@ -1145,14 +1145,44 @@ function FWVersion:minor() end
 ---@return integer
 function FWVersion:major() end
 
--- get field
+--get APM_BUILD_? value from AP_Vehicle/AP_Vehicle_Type.h that is checked against APM_BUILD_TYPE()
+-- here are some common ones:
+-- #define APM_BUILD_Rover          1
+-- #define APM_BUILD_ArduCopter     2
+-- #define APM_BUILD_ArduPlane      3
+-- #define APM_BUILD_AntennaTracker 4
+-- #define APM_BUILD_ArduSub        7
+-- #define APM_BUILD_AP_Periph      9
+-- #define APM_BUILD_Blimp          12
+-- #define APM_BUILD_Heli           13
 ---@return integer
```

Comment:
There is markup for return types:
```suggestion
---@return integer
---| '1' Rover
---| '2' ArduCopter
---| '3' ArduPlane
---| '4' AntennaTracker
---| '7' ArduSub
---| '9' AP_Periph
---| '12' Blimp
---| '13' Heli
```